.circleci/src/commands/@mobile-commands.yml

```diff
@@ -1,5 +1,5 @@
-# Build the ios app
-mobile-build-ios:
+# Bundle ios app and prepare to build/upload/release
+mobile-bundle-ios:
```

Comment:
Maybe name this `mobile-prepare-ios` to match android?

Reply:
> Very cool, so basically using manual bumps of the version number to trigger full app builds right? And normally just doing a codepush

@sliptype yes indeed!! i just posted details in the description.